toolkits/slack/evals/eval_chat.py

```diff
@@ -0,0 +1,1020 @@
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    user_messages = [
+        "Get the members of the #general channel",
+        "Get the members of the general channel",
+        "Get a list of people in the #general channel",
+        "Get a list of people in the general channel",
+        "Show me who's in the #general channel",
+        "Show me who's in the general channel",
+        "Who is in the #general channel?",
+        "Who is in the general channel?",
+    ]
+
+    for user_message in user_messages:
+        suite.add_case(
+            name=f"Get conversation members by name: {user_message}",
+            user_message=user_message,
+            expected_tool_calls=[
+                ExpectedToolCall(
+                    func=get_members_from_conversation_by_name,
+                    args={
+                        "conversation_name": "general",
+                    },
+                ),
+            ],
+            critics=[
+                BinaryCritic(critic_field="conversation_name", weight=1.0),
+            ],
+        )
+
+    suite.add_case(
+        name="Get conversation members by id",
+        user_message="Get the members of the conversation with id '1234567890'",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=get_members_from_conversation_by_id,
+                args={
+                    "conversation_id": "1234567890",
+                },
+            ),
+        ],
+        critics=[
+            BinaryCritic(critic_field="conversation_id", weight=1.0),
+        ],
+    )
+
+    return suite
+
+
+@tool_eval()
+def get_conversation_history_eval_suite() -> EvalSuite:
+    """Create an evaluation suite for tools getting conversations history."""
+    suite = EvalSuite(
+        name="Slack Chat Tools Evaluation",
+        system_message="You are an AI assistant that can interact with Slack to send messages and get information from conversations, users, etc.",
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    no_arguments_user_messages_by_conversation_name = [
+        "Get the history of the #general channel",
+        "Get the history of the general channel",
+        "list the messages in the #general channel",
+        "list the messages in the general channel",
+    ]
+
+    for user_message in no_arguments_user_messages_by_conversation_name:
+        suite.add_case(
+            name=f"Get conversation history by name: '{user_message}'",
+            user_message=user_message,
+            expected_tool_calls=[
+                ExpectedToolCall(
+                    func=get_conversation_history_by_name,
+                    args={
+                        "conversation_name": "general",
+                    },
+                ),
+            ],
+            critics=[
+                BinaryCritic(critic_field="conversation_name", weight=1.0),
```

Comment:
Would be good to add some critics for the different timestamp parameters. For example, this eval would pass if it predicted something as ridiculous as `oldest_datetime="1738-07-04 12:12:12"` and `oldest_datetime="1738-08-04 12:12:12"`.